__tests__/server.test.js

```diff
@@ -227,13 +234,82 @@ describe("server --> nc_news_test", () =>{
       });
       test("Returns status code 400 if injection is attempted via url", () => {
         return request(server)
-          .get("/api/articles/first; DROP table articles;")
+          .get("/api/articles/1; DROP table articles;")
           .expect(400)
           .then((result) => {
             expect(result.error.text).toEqual("Invalid URL");
           });
       });
+    });
 
+    describe("GET /api/articles/:article_id/comments", () => {
+      test("Returns status code 200 and correctly formatted and sorted body to a correct requests", () => {
+        const expectedObject = {
+          comment_id: expect.any(Number),
+          article_id: expect.any(Number),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.stringMatching(
+            /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}\w$/
+          ),
+          };
+
+        return request(server)
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .expect("Content-Type", /json/)
+          .then(({ body }) => {
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments.length).toBe(11);
+            expect(body.comments).toBeSortedBy("created_at", {
+              descending: true,
+            });
+            body.comments.forEach((comment) => {
+              expect(Object.keys(comment)).toHaveLength(6);
+              expect(comment).toMatchObject(expectedObject);
+              expect(comment.comment_id).toBeGreaterThan(0);
+              expect(comment.article_id).toBeGreaterThan(0);
```

Comment:
This should be more specific, you would expect this comment to have the article id of 1 otherwise it means the wrong comments have been returned